### PR TITLE
Binding adapter instead of LanguageSelectionAdapter

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

- replaced recycler view with BindableAdapter in Audio languages options
- **TODO** select radio button for tap on each item
- **TODO** update preferences
- **TODO** implement the same in App Language

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by BenHenning** [comment]
@FareesHussain why is this marked as 'do not merge'? Is this ready for review? If so, please remove that from the title & let me know so that I can take a review pass on it. If not, please mark this as a 'draft' PR until it's ready to go into review.

File: app/src/main/res/layout/audio_language_fragment.xml [hunk 1]
```diff
@@ -45,6 +51,7 @@
         android:background="@color/generalNavigationBackground"
         android:clipToPadding="false"
         android:overScrollMode="never"
+        app:list="@{viewModel.audioLanguagesLiveData}"
```
Comment (anandwana001) [inline app/src/main/res/layout/audio_language_fragment.xml]: `list` when you pass the list
`data` when you pass livedata

File: app/src/main/java/org/oppia/android/app/options/CheckBoxClickListener.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package org.oppia.android.app.options
+
+interface CheckBoxClickListener {
```
Comment (anandwana001) [inline app/src/main/java/org/oppia/android/app/options/CheckBoxClickListener.kt:3]: You are not using this anywhere. Need to implement somewhere to use it.

**Comment by FareesHussain** [comment]
I'll create a new PR as this appoarch does'nt work and i'll create a new pr with reference to Practice tab soon..

**Pull request closed without merging**